locale/template.txt

```diff
@@ -0,0 +1,20 @@
+# textdomain: moreores
+
+[moreores] loaded.=
```

Comment:
This does not appear anywhere in the code. Did you use the translation update script?

Reply:
Oh, strange. I copied it from the other translation files as all of them have it.
I'll remove that string immediately. Thank you.